core/src/main/java/io/grpc/internal/ManagedChannelImpl.java

```diff
@@ -1395,8 +1395,8 @@ public AbstractSubchannel createSubchannel(CreateSubchannelArgs args) {
     }
 
     private SubchannelImpl createSubchannelInternal(CreateSubchannelArgs args) {
-      // TODO(ejona): can we be even stricter? Like loadBalancer == null?
-      checkState(!terminated, "Channel is terminated");
+      // No new subchannel should be created after delayedTransport has been shutdown.
```

Comment:
s/delayedTransport/load balancer/ ? Since this is ideally from the perspective of the caller.

Reply:
ACK